File: src/main/kotlin/com/joshy/hywrapper/data/skyblock/GardenUpgrade.kt
package com.joshy.hywrapper.data.skyblock

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
enum class GardenUpgrade(val cleanName: String) {
    @SerialName("GROWTH_SPEED")
    GROWTH_SPEED("Crop Growth Speed"),
    @SerialName("YIELD")
    CROP_UPGRADE("Crop Upgrade"),
    @SerialName("PLOT_LIMIT")
    PLOT_LIMIT("Plot Limit")
}

File: typescript/src/data/skyblock/GardenUpgrade.ts
export enum GardenUpgrade {
    GROWTH_SPEED = "GROWTH_SPEED",
    CROP_UPGRADE = "YIELD",
    PLOT_LIMIT = "PLOT_LIMIT",
}

export const GardenUpgradeInfo: Record<GardenUpgrade, string> = {
    [GardenUpgrade.GROWTH_SPEED]: "Crop Growth Speed",
    [GardenUpgrade.CROP_UPGRADE]: "Crop Upgrade",
    [GardenUpgrade.PLOT_LIMIT]: "Plot Limit",
};
